### DISCUSSION TITLE
Multiple words in a tag

### DISCUSSION THREAD

**Discussion by todd-tech-writer** [squidfunk/mkdocs-material#4001] [Q&A] [answered]
Hello!

I was just curious if it was possible to include multiple words in a single tag. For example, when I create the tag, `example tag`, on `page1.md` and search for that page containing with the tag, nothing shows in the search results. But if I only type in `example`, the page pops right up in the search results.

I'm running mkdocs `8.2.8` for context.

Let me know if you need more information and I can create an example of what is happening in my actual content. 

Thanks!

**Comment by squidfunk** [comment, marked as answer]
Multiple-word tags should work as well. If you can create an issue with a minimal reproducible example, I can look into it.

**Reply by todd-tech-writer** [reply]
Appreciate it! Opened an issue here: https://github.com/squidfunk/mkdocs-material/issues/4004
